webapp/app/[locale]/tunnel/_components/form.tsx

```diff
@@ -10,14 +9,6 @@ import { useTunnelState } from '../_hooks/useTunnelState'
 import { NetworkSelectors } from './networkSelectors'
 import { TokenInput } from './tokenInput'
 
-const TransactionStatus = dynamic(
-  () =>
-    import('components/transactionStatus').then(mod => mod.TransactionStatus),
-  {
-    ssr: false,
-  },
-)
-
```

Comment:
technically speaking, this is used in line 100 🤔  Though it will be removed once we migrate Bitcoin to work with drawers (the redesign, which is on hold for that part). Maybe we can import it directly, instead of using dynamic?

Reply:
ESLint reported it as not used but you are right. Restoring that piece in the next commit.